src/shared_modules/keystore/src/keyStore.cpp

```diff
@@ -18,6 +18,12 @@ constexpr auto KS_NAME {"keystore"};
 constexpr auto KS_VERSION {"2"};
 constexpr auto KS_VERSION_FIELD {"version"};
 
+namespace Log
+{
+    std::function<void(const int, const char*, const char*, const int, const char*, const char*, va_list)>
+        GLOBAL_LOG_FUNCTION;
+};
+
```

Comment:
I understand the change, but was this strictly necessary for this implementation? It's ok if we are sure it will always work, I see that the indexerconnector testtool already initializes this variable.